### ISSUE TITLE
Changing fonts not working on Firefox

### ISSUE THREAD

**Issue by tadsn3w** [excalidraw/excalidraw#5110]
Using the text tool, and writing anything.  I tried to change the font to handwriting, it does not work.
It is always the normal font activated.
I refreshed, close firefox, but it issue remains the same.

I tried Brave and I can see the 3 different fonts.

**Comment by dwelle** [comment]
Hey, what's your Operating system and Firefox version?

**Comment by tadsn3w** [comment]
Firefox 99.0.1 64 bit.
Windows 10

**Comment by dwelle** [comment]
Cannot repro on Win 10, Firefox 100 & 85 (I could try 99 later, but I'm pretty sure it will work as I've used that version recently). Can you please record a video showing the exact problem (or write detailed steps)?

**Comment by ad1992** [comment]
@tadsn3w possible to try in firefox 100 and confirm if the bug is still reproducible?

**Comment by tadsn3w** [comment]
I installed the 100 version and it is the same result.

**Comment by tadsn3w** [comment]
I found the problem.  It is very simple and it is not excalidraw issue.
I had two 4k monitors 32" within Windows OS.  Windows cannot handle the 4K in all situations nor all the other software.  Icons too small, fonts ridiculously small.  I like to use the native resolution because the  3840x2160 gives a lot of vertical real state.
So, I changed font sizes in many programs.  Firefox is one of those.  I had unclicked (not selected the allow pages to choose their own font.) 
I put it back on and the problem went away.
I can change fonts without any problem.
![image](https://user-images.githubusercontent.com/44947400/166891185-9816a62e-afbc-4396-baf3-c13c984feffa.png)
